kinley-cody/scripts/article.js

```diff
@@ -65,6 +65,11 @@ Article.fetchAll = function() {
     // and then render the index page.
     $.get('data/hackerIpsum.json', function(data){
       localStorage.rawData = JSON.stringify(data);
+      Article.fetchAll();
```

Comment:
You should not be calling `Article.fetchAll()` inside `Article.fetchAll()` as it is recursion that will cause redundancy.